common/commands.h

```diff
@@ -118,7 +119,7 @@ namespace commands {
 
     public:
         pack::String address  = FIELD("address");
-        pack::String protocol = FIELD("protocol");
+        Protocol     protocol = FIELD("protocol");
```

Comment:
Please, don't change this one, need to define Protocol class in configuration manager to use in configuration